templates/scenarios/common/init-debug-vs-bot/teamsfx/app.local.yml

```diff
@@ -0,0 +1,46 @@
+version: 1.0.0
+
+registerApp:
+  - uses: teamsApp/create # Creates a Teams app
+    with:
+      name: {%appName%} # Teams app name
```

Comment:
Use `.tpl` file extension when using placeholder.

Reply:
Yes, this file is just a placeholder, you can update it if necessary